CHANGELOG.md

```diff
@@ -1,3 +1,823 @@
+  [#16347](https://github.com/angular/angular.js/issues/16347))
+- **input[number]:** validate min/max against viewValue
+  ([aa3f95](https://github.com/angular/angular.js/commit/aa3f951330ec7b10b43ea884d9b5754e296770ec),
+  [#12761](https://github.com/angular/angular.js/issues/12761),
+  [#16325](https://github.com/angular/angular.js/issues/16325))
+- **$httpParamSerializerJQLike:**
+  - call functions as jQuery does
+  ([a784fa](https://github.com/angular/angular.js/commit/a784fab605d825f1158c6292b3c42f8c4a502fdf),
+  [#16138](https://github.com/angular/angular.js/issues/16138),
+  [#16139](https://github.com/angular/angular.js/issues/16139))
+  - follow jQuery for `null` and `undefined`
+  ([301fdd](https://github.com/angular/angular.js/commit/301fdda648680d89ccab607c413a7ddede7b0165))
+- **$parse:**
+  - do not pass scope/locals to interceptors of one-time bindings
+  ([87a586](https://github.com/angular/angular.js/commit/87a586eb9a23cfd0d0bb681cc778b4b8e5c8451d))
+  - always pass the intercepted value to watchers
+  ([2ee503](https://github.com/angular/angular.js/commit/2ee5033967d5f87a516bad137686b0592e25d26b),
+  [#16021](https://github.com/angular/angular.js/issues/16021))
+  - respect the interceptor.$stateful flag
+  ([de7403](https://github.com/angular/angular.js/commit/de74034ddf6f92505ccdb61be413a6df2c723f87))
+- **Angular:** remove `angular.lowercase` and `angular.uppercase`
+  ([1daa4f](https://github.com/angular/angular.js/commit/1daa4f2231a89ee88345689f001805ffffa9e7de),
+  [#15445](https://github.com/angular/angular.js/issues/15445))
+- **$controller:** remove instantiating controllers defined on window
+  ([e269c1](https://github.com/angular/angular.js/commit/e269c14425a3209040f65c022658770e00a36f16),
+  [#15349](https://github.com/angular/angular.js/issues/15349),
+  [#15762](https://github.com/angular/angular.js/issues/15762))
+
+
+## New Features
+- **isArray:** support Array subclasses in `angular.isArray()`
+  ([e3ece2](https://github.com/angular/angular.js/commit/e3ece2fad9e1e6d47b5f06815ff186d7e6f44948),
+  [#15533](https://github.com/angular/angular.js/issues/15533),
+  [#15541](https://github.com/angular/angular.js/issues/15541))
+- **$sce:** handle URL sanitization through the `$sce` service
+  ([1e9ead](https://github.com/angular/angular.js/commit/1e9eadcd72dbbd5c67dae8328a63e535cfa91ff9))
+- **orderBy:** consider `null` and `undefined` greater than other values
+  ([1d8046](https://github.com/angular/angular.js/commit/1d804645f7656d592c90216a0355b4948807f6b8),
+  [#15294](https://github.com/angular/angular.js/issues/15294),
+  [#16376](https://github.com/angular/angular.js/issues/16376))
+- **$resource:** add support for `request` and `requestError` interceptors (#15674)
+  ([240a3d](https://github.com/angular/angular.js/commit/240a3ddbf12a9bb79754031be95dae4b6bd2dded),
+  [#5146](https://github.com/angular/angular.js/issues/5146))
+- **$rootScope:** allow suspending and resuming watchers on scope
```

Comment:
Looks like this was forgotten to backport! Since it's a completely new API, there should be no problem with BC